src/main/java/io/kontur/disasterninja/config/WebConfiguration.java

```diff
@@ -36,15 +42,32 @@
 import java.util.concurrent.TimeUnit;
 
 @Configuration
+@RequiredArgsConstructor
 public class WebConfiguration {
 
+    private final MeterRegistry meterRegistry;
+
+    @Bean
+    public HttpClient httpClient() {
```

Comment:
```suggestion
    @Bean
    public HttpClient httpClient(MeterRegistry meterRegistry) {
```

Reply:
You can inject meterRegistry into method and remove @RequiredArgsConstructor annotation

Reply:
Thought about that. But since `httpClient()` is called while creating each `RestTemplate`, you'll need to explicitly provide `meterRegistry` in such case

Reply:
But you don't have to call it explicitly. Add `HttpClient httpClient` to each method. You've already registered Bean in context.